benchmark/C/Savina/src/parallelism/PiPrecision.lf

```diff
@@ -196,13 +189,24 @@ reactor Worker {
     input doWork:uint32_t;
     output response:mpf_t;
     
+    reaction(startup) {=
```

Comment:
Each worker will now initialize the global variables (potentially in parallel). This doesn't look safe...